apps/core/.solcover.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    skipFiles: ["trident", "core/interfaces", "core/mocks"],
```

Comment:
```suggestion
    skipFiles: ["core/interfaces", "core/mocks"],
```

We are not longer using trident so this can be removed.